src/wiki/common-misconceptions.md

```diff
@@ -18,12 +18,30 @@ RAM utilization is also dependent on your RAM capacity - the more RAM you have,
+The improvement is not real, if marginal at best. Uninstalling the pre-installed apps, disabling system services, and messing with the registry will 99% of the time **not** bring you an improvement in performance. Windows does not load up all of those apps in the memory at the same time, only when you use them. Services take up a very insignificant amount of memory and CPU resources, even on a low end machine. And Windows does not access the registry sequentially, meaning that deleting registry keys using dubious third party cleaners such as CCleaner will not bring about an improvement as Windows accesses that key directly&mdash;it is not akin to disk fragmentation, and no improvement will be gained from similarly dubious registry tweaks.
 
 Attempting to 'debloat' Windows will most of the time bring about problems instead of improvements - it has been tested time and time again and the results show that the default state of Windows will not harm performance. Unless you have an OEM machine with actual OEM bloatware ([that can and will harm performance](https://www.youtube.com/watch?v=5N7aYtkzKJc)) installed on it, then a clean installation of Windows is recommended. But **attempting to debloat an already clean installation will give you more trouble than it's worth**.
 
 Besides, if you are concerned about these things affecting your gaming performance, Windows already has its own 'Game Mode' built in that works quite well at suppressing its background tasks while you play. It's also enabled by default, so you don't need to do anything more.
 
+### OEM Bloatware
+
+While it is recommended to perform a clean install of Windows as the best way to get rid of bloatware, some devices and motherboard manufacturers will automatically re-install these immediately after you finish setup. This practice has seen some controversy, notably the [Lenovo SuperFish vulnerability in 2014](https://support.lenovo.com/us/en/product_security/ps500035-superfish-vulnerability) and [again in 2015](https://www.howtogeek.com/226308/the-windows-platform-binary-table-why-crapware-can-come-back-after-a-clean-install/).
+
+The good news is that these can usually be disabled, with some tradeoffs such as no automatic driver installation or, if any, RGB LED control.
+
+A list of brands with OEM bloatware and how you can disable them is compiled below. That being said, BIOS updates may change these settings, so it is a good idea to double-check occasionally.
+
+* ASUS / ROG
+  * Armoury Crate: https://www.asus.com/support/faq/1043788/
+  * On some models: *ASUS GRID install service* in BIOS
+* MSI: *MSI Driver Utility Installer* in BIOS
+* ASRock: *Auto Driver Installer* in BIOS
+* Gigabyte: *Gigabyte Utilities Downloader Configuration* in BIOS.
```

Comment:
```suggestion
  * Armoury Crate: https://www.asus.com/support/faq/1043788/
  * On some models: *ASUS GRID install service* in BIOS.
* MSI: *MSI Driver Utility Installer* in BIOS.
* ASRock: *Auto Driver Installer* in BIOS.
* Gigabyte: *Gigabyte Utilities Downloader Configuration* in BIOS.
```

Reply:
The bullet points aren't full sentences so  depending who you ask it wouldn't be grammatically correct to put periods on them. That said, it is strange one of them has it but not the other.

Reply:
This might've been a typo. I had this section formatted differently before I settled on this one